datalab/datalab_session/data_operations/median.py

```diff
@@ -31,4 +37,50 @@ def wizard_description():
         }
     
     def operate(self):
-        pass
+        input_files = self.input_data.get('input_files', [])
+        completion_total = len(input_files)
+        image_data_list = []
+
+        log.info(f'Executing median operation on {completion_total} files')
+
+        # fetch fits for all input data
+        for index, file_info in enumerate(input_files):
+            basename = file_info.get('basename', 'No basename found')
+            
+            fits_file = find_fits(basename)
```

Comment:
find_fits isn't returning a fits file, it's actually returning an archive record which points to a FITS file, so we should rename this